source/controllers/src/impedance/Dissipative.cpp

```diff
@@ -1,62 +1,17 @@
 #include "controllers/impedance/Dissipative.hpp"
 
+using namespace StateRepresentation;
```

Comment:
Yes so the problem is because those are templates. When defined in header files you don't need to specify the specialized class that can work with this implementation. The compiler simply assumes and tries to compile irrelevant of the input class of the template. Conversely, when defined in source file, you need to manually specify which class are working and provides specialization for each of them. You can defined a generic template function and specify which input class are accepted or defined specialized function for each of them.